src/services/user-service.js

```diff
@@ -10,8 +10,8 @@ import PasswordResetRequestStore from '../stores/password-reset-request-store.js
 const DEFAULT_ROLE_NAME = ROLE.user;
 const PROVIDER_NAME = 'elmu';
 const PASSWORD_SALT_ROUNDS = 1024;
-const PENDING_USER_REGISTRATION_EXPIRATION_IN_HOURS = 24;
-const PENDING_PASSWORD_RESET_REQUEST_EXPIRATION_IN_HOURS = 24;
+const PENDING_USER_REGISTRATION_EXPIRATION_IN_HOURS = { hours: 24 };
+const PENDING_PASSWORD_RESET_REQUEST_EXPIRATION_IN_HOURS = { hours: 24 };
```

Comment:
These 2 guys could now drop the `_IN_HOURS` suffix.

Reply:
...maybe replacing it with `_TIMESPAN` (to make it clearer that it is an object not a scalar value)?

Reply:
done